docs/Configuring-Orion/Using-Oracle.md

```diff
@@ -0,0 +1,59 @@
+description: Storing Information with Oracle
+<!--- END of page meta data -->
+
+# Store Information with Oracle
+
+Oracle is an object-relational database you can use to store private transaction payloads for Orion. Use a relational database to support your [disaster recovery strategy](Disaster-Recovery.md).
+
+## Prerequisites
+
+* [Oracle Database](https://docs.oracle.com/en/)
+
+## Set up your Oracle database
+
+In your database:
+
+1. Create the `store` table by using the provided DDL script, as shown in the following example. Substitute your own values for `HOST-NAME`, `USER-NAME`, and `DATABASE-NAME`.
+
+    `sqlplus -h <HOST-NAME> -U <USER-NAME> -d <DATABASE-NAME> -f ~<INSTALLATION_DIRECTORY>/orion/database/oracle_ddl.sql` 
```

Comment:
There are different ways to run scripts with sqlplus. E.G:
sqlplus username/password@databasename @script

Reply:
Did you get this question answered @sgregglives?

Reply:
The sqlplus command depends a lot on the version that is used. 
I think that version 11 and newer support this:
sqlplus username/password@host:port/service
or if you want to use the sid instead of the service you can do this:
sqlplus username/password@host:port:sid
There seem to be a lot of ways to connect to a DB with sqlplus ....
People using Oracle should know how they can connect to their DB!
Maybe best if we just give an example in the docs and we rely on the fact that people creating tables on Oracle DBs know how to connect to it.

Reply:
Thanks for this information. I understand the situation now.